.github/workflows/code-climate-coverage-aggregation.yml

```diff
@@ -84,6 +84,10 @@ jobs:
       - name: Run Yarn Prepare - Shared components
         run: yarn --cwd src/frontend/shared-components prepare
 
+      # Run Yarn Coverage to generate javascript coverage report
+      - name: Run Yarn Coverage - Shared components
+        run: yarn --cwd /home/runner/work/jag-file-submission/jag-file-submission/src/frontend/shared-components coverage
```

Comment:
/home/runner/work/jag-file-submission/jag-file-submission/src/frontend/shared-components

should be

${{ github.workspace }}/src/frontend/shared-components